ansible/roles_ocp_workloads/ocp4_workload_5gran_deployments_lab/tasks/workload.yml

```diff
@@ -203,6 +205,27 @@
     - argocd_app_sno1_deployment.resources[0].status.health.status == "Healthy"
     - argocd_app_sno1_deployment.resources[0].status.sync.status == "Synced"
 
+# HAproxy was installed in pre_workload
+- name: Ensure HAProxy can listen on port 6443
+  community.general.seport:
+    ports: 6443
+    proto: tcp
+    setype: http_port_t
+    state: present
+
+- name: Ensure HAProxy config is downloaded
+  ansible.builtin.get_url:
+    url: "https://raw.githubusercontent.com/RHsyseng/5g-ran-deployments-on-ocp-lab/main/lab-materials/lab-env-data/haproxy/haproxy.cfg"
```

Comment:
same question

Reply:
We plan to do that once we test this on dev, next step will be adding a var to control the content.

Reply:
version URL included